solr/src/main/scala/akka/stream/alpakka/solr/javadsl/SolrFlow.scala

```diff
@@ -22,29 +23,41 @@ object SolrFlow {
    * Java API: creates a [[akka.stream.alpakka.solr.SolrFlowStage]] for [[SolrInputDocument]]
    * from [[IncomingMessage]] to sequences of [[IncomingMessageResult]].
    */
-  def document(
+  def documents(
```

Comment:
Please keep the original signatures and complement with the `Seq`-versions (by putting the single element in a Seq).